framework/test/jest/unit/specs/modules/chain/dpos/apply.spec.js

```diff
@@ -60,6 +60,11 @@ describe('dpos.apply()', () => {
 			error: jest.fn(),
 		};
 
+		stubs.channel = {
+			subscribe: jest.fn(),
+			publish: jest.fn(),
```

Comment:
now publishing event is part of behaviour of `apply` and `undo`. So, could you please add one test case per each method to check if stubs are called with right parameters if the block is the last block of the round.

Reply:
`channel` was replaced by `events` and appropriate tests were added.